docs/reference/src/main/asciidoc/extend.asciidoc

```diff
@@ -138,54 +137,18 @@ An extension observer method is not allowed to inject any other object.
 
 === The `BeanManager` object
 
-The nerve center for extending CDI is the `BeanManager` object. The
-`BeanManager` interface lets us obtain beans, interceptors, decorators,
-observers and contexts programmatically.
+The nerve center for extending CDI is the `BeanManager` object.
+The `BeanManager` interface provides operations of use to portable extensions, e.g. lets us obtain beans, interceptors, decorators, observers and contexts programmatically.
```

Comment:
Well, I tried to follow the spec wording. But you're right that it sounds odd to my Czech ears ;-) 
Maybe `The BeanManager interface provides operations useful for portable extensions`?